src/main/java/in/projecteka/consentmanager/link/Constants.java

```diff
@@ -5,13 +5,17 @@
 public class Constants {
     public static final String APP_PATH_CONFIRM_LINK = "/v1/links/link/confirm";
     public static final String APP_PATH_CONFIRM_LINK_REF_NUMBER = APP_PATH_CONFIRM_LINK + "/{linkRefNumber}";
+    public static final String APP_PATH_CARE_CONTEXTS_DISCOVER = "/v1/care-contexts/discover";
+    public static final String APP_PATH_INTERNAL_GET_LINKED_CARE_CONTEXTS = "internal/patients/{username}/links";
+    public static final String APP_PATH_LINK_INIT = "/v1/links/link/init";
+    public static final String APP_PATH_GET_PATIENTS_LINKS = "/patients/links";
 
```

Comment:
I think we are using /v0.5 for these url's in gateway

Reply:
Not for the apps.

Reply:
okay.